Sources/PackageCollections/PackageCollections.swift

```diff
@@ -294,6 +354,35 @@ public struct PackageCollections: PackageCollectionsProtocol {
             return PackageCollectionsModel.TargetListItem(target: pair.target, packages: targetPackages)
         }
     }
+
+    internal static func mergedPackageMetadata(package: PackageCollectionsModel.Collection.Package,
+                                               basicMetadata: PackageCollectionsModel.PackageBasicMetadata?) -> PackageCollectionsModel.Package {
+        var versions = package.versions.map { packageVersion in
+            PackageCollectionsModel.Package.Version(version: packageVersion.version,
+                                                    packageName: packageVersion.packageName,
+                                                    targets: packageVersion.targets,
+                                                    products: packageVersion.products,
+                                                    toolsVersion: packageVersion.toolsVersion,
+                                                    verifiedPlatforms: packageVersion.verifiedPlatforms,
+                                                    verifiedSwiftVersions: packageVersion.verifiedSwiftVersions,
+                                                    cves: nil, // TODO: where do we get this from?
```

Comment:
How about we remove this and revisit in the future?

Reply:
addressed by adding it to the model, we can refactor further as we implement the actual providers